.github/workflows/release.yaml

```diff
@@ -1,8 +1,12 @@
 name: Helm Charts
 on:
-  push:
+  workflow_run:
+    workflows:
+      - "Test Chart"
     branches:
       - main
+    types:
+      - completed
```

Comment:
From my understanding, this does not mean successful, just that it finished.

You need to add this to the job steps:
```
if: ${{ github.event.workflow_run.conclusion == 'success' }}
```
from my understanding.

Reply:
Yes, you're right: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-a-workflow-based-on-the-conclusion-of-another-workflow